test/integration/cors-proxy-test.js

```diff
@@ -1,18 +1,14 @@
 var assert = require('chai').assert
-var supertest = require('supertest')
 var path = require('path')
 var nock = require('nock')
-var { checkDnsSettings } = require('../utils')
-
-var ldnode = require('../../index')
+var { checkDnsSettings, setupSupertestServer } = require('../utils')
 
 describe('CORS Proxy', () => {
-  var ldp = ldnode({
+  var server = setupSupertestServer({
     root: path.join(__dirname, '../resources'),
```

Comment:
Well, if you want to go cross-platform, `path.join(__dirname, '..', 'resources')` would make Windows users happy.

Reply:
> Well, if you want to go cross-platform, `path.join(__dirname, '..', 'resources')` would make Windows users happy.

Is it necessary though? It seems like [path.join will normalize it nevertheless](https://shapeshed.com/writing-cross-platform-node/#use-path-join-to-join-folder-names).

Reply:
Hm, you made me look at the [source code of `path.join`](https://github.com/nodejs/node/blob/v11.2.0/lib/path.js#L488).

The call to [`win32.normalize`](https://github.com/nodejs/node/blob/v11.2.0/lib/path.js#L284) does some … things. So it could be possible that it would normalise it on its own.